docs/trn-generation.md

```diff
@@ -0,0 +1,31 @@
+# Teaching Reference Number (TRN) Generation
+
+TRN generation is currently performed by 3 separate organisations, DfE, EWC, and Teaching Pensions. This will change from 2024 when we phase in a new TRN generation flow. This will have the following characteristics:
+
+* The Department for Education (DfE) will become the only issuer of TRN's
+* TRN generation will be RESTFUL API based only (no file generation)
+* Calling the API will return one of three possible successful outcomes:
+  *   TRN generated and returned
```

Comment:
This is a little different to the current spec; either you'll be returned a TRN or you'll get a 'Pending' status and have to callback later. There's no way of knowing from the response whether the TRN already existed or was just created.